### ISSUE TITLE
Annotation review O75909 (CCNK_HUMAN)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#2090]
negative regulation of cell cycle arrest

I didn't read the paper but I wanted to flag this one. 

If this refers to the phenotype  of "arrested cells"
"negative regulation of arrest" isn't  the correct process......

Normal arrest happens in response to checkpoints The arrest caused by ablating a cyclin
will be causing issues that are detected by the checkpoints.

In addition, this appears to be a complementation assay with cerevisiae. This would be dodgy to infer anything about the type of cell cycle regulation because ectopically expressed cyclins can  replace each other when expressed at  the wrong time point.
So this is rescuing G1 cyclin defects in S. cerevisiae, but in reality it is likely to be a transcription elongation cyclin.

In summary, I just wouldn't make an annotation from this.

**Comment by pgaudet** [comment]
This is a MGI annotation @hdrabkin

**Comment by hdrabkin** [comment]
removed.
